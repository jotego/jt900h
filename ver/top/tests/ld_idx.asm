    ; Some indexed addressing
    main section code
    org 0
    ld a,5
    ld xix,data
    ld wa,(data)
    ld bc,(data+2)
    ld de,(xix)
    nop
    nop
    nop
    nop
    nop
    nop
forever:
    jp forever
data:
    dw 0xcafe,0xbeef,0x1234,0x6789,0xabcd
    end