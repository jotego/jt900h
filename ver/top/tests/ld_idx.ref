0000cafe - 00000000 - 00000000 - 00000000
0000beef - 00000000 - 00000000 - 00000000
0000cafe - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
-- index registers --
00000022 - 00000000 - 00000000 - 00000100
SR = f800
