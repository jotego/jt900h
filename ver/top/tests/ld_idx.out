0000cafe - 00000000 - 00000000 - 00000000
0000beef - 00000000 - 00000000 - 00000000
0000cafe - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
-- index registers --
0000001d - 00000000 - 00000000 - 00000000
SR = 00f8
